web-common/src/features/dashboards/state-managers/selectors/measures.ts

```diff
@@ -175,6 +175,18 @@ export const getIndependentMeasures = (
   return [...measures];
 };
 
+export const getFilteredSimpleMeasures = (
```

Comment:
Nit: rename to `getSimpleMeasures()`. `getFilteredSimpleMeasures()` almost sounds like the simple measures are somehow further filtered.